java/arcs/core/entity/EntityBase.kt

```diff
@@ -134,6 +134,9 @@ open class EntityBase(
     /** Returns the [FieldType] for the given singleton field, or null if it does not exist. */
     private fun getSingletonTypeOrNull(field: String) = schema.fields.singletons[field]
 
+    /** Returns true if the singleton has the [FieldType]. */
```

Comment:
Maybe clearer to say "returns true if the singleton has the Field"

Reply:
👍